ndc-reference/tests/query/table_argument_order_by/expected.json

```diff
@@ -2,10 +2,10 @@
   {
     "rows": [
       {
-        "id": 2
+        "id": 1
       },
       {
-        "id": 1
+        "id": 2
```

Comment:
Is this relevant to the casing change?

Reply:
No, good catch. I'll look into this, thanks.